### PR TITLE
version bump

### PR DESCRIPTION
**Pull request by lei0zhou**
this pr is the implementation for #16

### CODE REVIEW

**Comment by jc00ke** [comment]
I'll try to find some laptop time to look at this.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -37,8 +41,8 @@ jobs:
            fetch-depth: 0
       - uses: erlef/setup-beam@v1
         with:
-          otp-version: '25.3.1'
-          elixir-version: '1.14.5'
+          otp-version: '27.0.1'
```
Comment (jc00ke) [inline .github/workflows/main.yml:44]: https://github.com/erlef/setup-beam/?tab=readme-ov-file#version-file

File: flake.nix [hunk 2]
```diff
@@ -21,7 +21,7 @@
         # erlang = pkgs.beam.interpreters.erlang;
 
         # use latest version of Erlang 27
-        erlang = final.beam.interpreters.erlang_25;
+        erlang = final.beam.interpreters.erlang_27;
```
Comment (jc00ke) [inline flake.nix:24]: This might have been what I needed when I was playing around with devenv!

**Review by jc00ke — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (lei0zhou) [reply .github/workflows/main.yml]: added

**Pull request merged**